Documentation/Getting-Started/glossary.md

```diff
@@ -0,0 +1,130 @@
+# Glossary
+
+## Rook
+
+### CephBlockPool CRD
+
+The [CephBlockPool CRD](../CRDs/Block-Storage/ceph-block-pool-crd.md) is used by Rook to allow creation and customization of storage pools.
+
+### CephBlockPoolRadosNamespace CRD
+
+CephBlockPoolRados Namespace CRD is used by Rook to allow creation of Ceph BlockPool RadosNamespaces. For more information and examples refer to this [documentation](../CRDs/Block-Storage/ceph-block-pool-rados-namespace-crd.md).
+
+### CephClient CRD
+
+CephClient CRD is used by Rook to allow [creation](https://rook.io/docs/rook/latest/CRDs/ceph-client-crd/) and updating clients.
+
+### CephCluster CRD
+
+CephCluster CRD is used by Rook to allow creation and customization of storage clusters through the custom resource definitions (CRDs). For more information about the different modes of creation of clusters, refer to our [documentation](../CRDs/Cluster/ceph-cluster-crd.md).
```

Comment:
For the CRDs, how about just putting a link on the CRD name like this? Then we don't need a separate generic statement about referring to the docs.
```suggestion
The [CephCluster CRD](../CRDs/Cluster/ceph-cluster-crd.md) is used by Rook to allow creation and customization of storage clusters through the custom resource definitions (CRDs).
```